test/data/testSP.py

```diff
@@ -347,6 +347,25 @@ def testmetadata_package_version(self):
             packaging.version.parse(self.data.metadata["package_version"]),
             packaging.version.Version
         )
+    
+    @skipForParser('ADF', 'reading input file contents and name is not implemented')

```

Comment:
Can you update the reason to all of these to something like 'reading cpu/wall times is not implemented for this parser.'

Reply:
Also can you add ORCA to this list. 

```
python -m pytest -v --capture=no --cov=cclib --cov-report=term --cov-report=xml:coverage_unit.xml test/test_data.py
```
If I run that locally with your branch the test is showing up and passing for QChem as expected.

Reply:
These should both be addressed now. Thanks for checking on the tests, I'll trouble shoot this locally then.